%YAML:1.0

image_L_topic: "/mynteye/left/image_raw"
image_R_topic: "/mynteye/right/image_raw"
ref_pose_topic: "/vins_estimator/ref_pose"
cur_pose_topic: "/vins_estimator/cur_pose"

image_width: 752
image_height: 480
T_BCL: !!opencv-matrix
  cols: 4
  rows: 4
  dt: d
  data: [-0.00646620000000000, -0.99994994000000004, -0.00763565000000000, 0.00000533646000000,
            0.99997908999999996, -0.00646566000000000, -0.00009558000000000, -0.00004302922000000,
            0.00004620000000000, -0.00763611000000000,  0.99997084000000003,  0.00002303124000000,
            0.0, 0.0, 0.0, 1.0]

intrinsics_L: [356.94410213878677496, 357.40888097897311582, 381.64638158380381583, 227.06518770187651057] #fu, fv, cu, cv
distortion_coefficients_L: [-0.25578212099603281, 0.04705855225214325, 0.00070597902035026, -0.00006641342704396]

T_BCR: !!opencv-matrix
  cols: 4
  rows: 4
  dt: d
  data: [-8.71864725e-03,  -9.99878610e-01,  -1.29131145e-02, -0.00168091910,
            9.98225224e-01,  -7.94201138e-03,  -5.90196631e-02, 0.120598251,
            5.89099369e-02,  -1.34047698e-02,   9.98173294e-01, 0.00153429370,
            0.0, 0.0, 0.0, 1.0]

intrinsics_R: [360.26500808807332987, 361.00061286165828278, 368.33314446451811364, 233.24107413011483914] #fu, fv, cu, cv
distortion_coefficients_R: [-0.26040909867113921, 0.04667523101515141, 0.00099042424008858, -0.00061525011270181]

baseline: 0.12 # (mm)

num_disparities: 64
P1: 16
P2: 64
Gtau: 8
Q1: 1
Q2: 1
downsample: 1
